client/x/evmstaking/keeper/delegator_address.go

```diff
@@ -148,7 +148,6 @@ func (k Keeper) ProcessRemoveOperator(ctx context.Context, ev *bindings.IPTokenS
 				types.EventTypeRemoveOperatorFailure,
 				sdk.NewAttribute(types.AttributeKeyBlockHeight, strconv.FormatInt(sdkCtx.BlockHeight(), 10)),
 				sdk.NewAttribute(types.AttributeKeyDelegatorUncmpPubKey, hex.EncodeToString(ev.UncmpPubkey)),
-				sdk.NewAttribute(types.AttributeKeyOperatorAddress, ev.Operator.Hex()),
```

Comment:
Why do we remove this attribute

Reply:
new event does not include `operator_address`, we could get from our KVS but imo it's useless as we only support single operator address and it's failure log